clusters/openstack/icehouse/cluster.build.properties

```diff
@@ -0,0 +1,2 @@
+cluster.pan.includes=sites/example openstack/icehouse os quattor/14.6.0 standard 
```

Comment:
The 14.6.0 (or earlier) has been successfully tested. Newer version of Quattor are not working (issue with ncm-network component). It is fixed in 15.4 release (see https://github.com/quattor/configuration-modules-core/pull/423).